server/index.ts

```diff
@@ -1,6 +1,6 @@
 import path from 'path';
 import dotEnv from 'dotenv';
-import express, { Request, Response } from 'express';
+import express, { NextFunction, Request, Response } from 'express';
```

Comment:
Doesn't look like NextFunction is being used at the moment, but maybe intended for line 138?